include/rcutils/get_env.h

```diff
@@ -61,6 +61,30 @@ RCUTILS_WARN_UNUSED
 const char *
 rcutils_get_env(const char * env_name, const char ** env_value);
 
+// Retrieve the full path to the home directory.
+/*
```

Comment:
You have to use `/**` if you want doxygen to pick it up (in my experience), but it would also be more consistent with our other docs.

Reply:
I decided to keep consistent with how the rest of the documentation in rcutils is.  If we want to improve it, I think we should do a pass over all of the header files together.

Reply:
No, we definitely use `/**` more than `/*`, e.g.:

https://github.com/ros2/rcutils/blob/e73b2167904d1fa349cb96e325297caa595a636d/include/rcutils/format_string.h#L30
https://github.com/ros2/rcutils/blob/e73b2167904d1fa349cb96e325297caa595a636d/include/rcutils/filesystem.h#L31
https://github.com/ros2/rcutils/blob/e73b2167904d1fa349cb96e325297caa595a636d/include/rcutils/error_handling.h#L102
https://github.com/ros2/rcutils/blob/e73b2167904d1fa349cb96e325297caa595a636d/include/rcutils/find.h#L27

We also use `/**` pretty much everywhere, e.g. in `rcl` and `rclcpp`.

Reply:
OK, got it.  I guess I was just looking at get_env.h, which is the one place we don't do this :).  I'll fix it up.